### ISSUE TITLE
glitchsoc-sidekiq: postfix/postdrop warnings (mails not sent)

### ISSUE THREAD

**Issue by junochat** [YunoHost-Apps/glitchsoc_ynh#30]
Current version: 2022.01.31~ynh.
Mails sent by Glitch-soc (such as sign in security/authentication verification needed to login) are no longer sent out. No errors from `postfix` logs. From my understanding of Mastodon, mails are handled through `glitchsoc-sidekiq` first.

```
yunohost service log glitchsoc-sidekiq
  - Feb 14 07:52:49 bundle[4623]: postdrop: warning: mail_queue_enter: create file maildrop/708919.22143: Permission denied
  - Feb 14 07:52:49 postfix/postdrop[22143]: warning: mail_queue_enter: create file maildrop/708919.22143: Permission denied
  - Feb 14 07:52:59 bundle[4623]: postdrop: warning: mail_queue_enter: create file maildrop/709223.22143: Permission denied
  - Feb 14 07:52:59 postfix/postdrop[22143]: warning: mail_queue_enter: create file maildrop/709223.22143: Permission denied
  - Feb 14 07:53:09 bundle[4623]: postdrop: warning: mail_queue_enter: create file maildrop/709442.22143: Permission denied
  - Feb 14 07:53:09 postfix/postdrop[22143]: warning: mail_queue_enter: create file maildrop/709442.22143: Permission denied
  - Feb 14 07:53:19 bundle[4623]: postdrop: warning: mail_queue_enter: create file maildrop/709658.22143: Permission denied
  - Feb 14 07:53:19 postfix/postdrop[22143]: warning: mail_queue_enter: create file maildrop/709658.22143: Permission denied
  - Feb 14 07:53:29 bundle[4623]: postdrop: warning: mail_queue_enter: create file maildrop/709874.22143: Permission denied
  - Feb 14 07:53:29 postfix/postdrop[22143]: warning: mail_queue_enter: create file maildrop/709874.22143: Permission denied
  - Feb 14 07:53:39 bundle[4623]: postdrop: warning: mail_queue_enter: create file maildrop/710099.22143: Permission denied
  - Feb 14 07:53:39 postfix/postdrop[22143]: warning: mail_queue_enter: create file maildrop/710099.22143: Permission denied
```

Restoring version 2021.12.27~ynh2 , mails are still successfully sent.
May be a similar issue to https://github.com/YunoHost-Apps/mastodon_ynh/issues/305

Reading through this issue, commenting out lines added https://github.com/YunoHost-Apps/glitchsoc_ynh/commit/85f56d463bea08ff480203b126eeb83f02c36fdf#diff-6a55ee521c9f673db2c3bbccf561ad51156d4cb5eb8391f934d096d33605be2f seems to fix mails problem as well.

### Context
Hardware: VPS
Yunohost version: 4.3.6.2
Glitch-soc version: 2022.01.31~ynh1 update through CLI

**Comment by Tagadda** [comment]
Thanks for the report! It's definitely the same issue as mastodon_ynh.
I'm looking into it.

**Comment by Tagadda** [comment]
This should be fix by [ebf1814](https://github.com/YunoHost-Apps/glitchsoc_ynh/commit/ebf1814d5a01c7d070d9c4e532c9b2053ffb6acc).
You can use to the `testing` branch with `sudo yunohost app upgrade glitchsoc -u https://github.com/YunoHost-Apps/glitchsoc_ynh/tree/testing`
